jit/v2/JIT_v2.0.0.yaml

```diff
@@ -341,6 +756,102 @@ paths:
           headers:
             API-Version:
               $ref: '#/components/headers/API-Version'
+        '400':
+          description: |
+            In case omitting a `Transport Call` does not schema validate a `400` (Bad Request) is returned
```

Comment:
In case omitting a 'Terminal Call' fails schema validation a 400 (Bad Request) is returned